Common/Data/Model/Course.swift

```diff
@@ -78,6 +78,13 @@ public final class Course: NSManagedObject {
         return self.enrollment != nil && self.enrollment?.objectState != .deleted
     }
 
+    public var openCourseUserActivity: NSUserActivity {
+        let userActivity = NSUserActivity(activityType: Bundle.main.bundleIdentifier!.appending("openCourse"))
+        userActivity.title = title! + ": " + url!.absoluteString
```

Comment:
Just the course title should be fine https://developer.apple.com/documentation/foundation/nsuseractivity/1413375-title